tests/test_evaluator.py

```diff
@@ -1,347 +1,77 @@
+                                                            start_position=127,
+                                                            end_position=134),
+                                        predicted_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=124,
+                                                            end_position=134),
+                                        overlap_score=0.82),
+                             SpanOutput(output_type="EXACT",
+                                        annotated_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=164,
+                                                            end_position=174),
+                                        predicted_span=Span(entity_type="PER",
+                                                            entity_value="",
+                                                            start_position=164,
+                                                            end_position=174),
+                                        overlap_score=1),
+                             SpanOutput(output_type="STRICT",
+                                        annotated_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=197,
+                                                            end_position=205),
+                                        predicted_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=197,
+                                                            end_position=205),
+                                        overlap_score=1),
+                             SpanOutput(output_type="STRICT",
+                                        annotated_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=208,
+                                                            end_position=219),
+                                        predicted_span=Span(entity_type="LOC",
+                                                            entity_value="",
+                                                            start_position=208,
+                                                            end_position=219),
+                                        overlap_score=1),
+                             SpanOutput(output_type="PARTIAL",
+                                        annotated_span=Span(entity_type="MISC",
```

Comment:
I think that's a good suggestion. Currently, we only have one function in this test file. However, if we need to implement another function that reuses these spans, I will define them at the top of the test.

Reply:
What I mean is to re-use the already defined span in this test instead of creating it twice. Something like:
```python
expected_span_outputs = [SpanOutput(output_type="ENT_TYPE",
                                        annotated_span=annotated_spans[1],
                                        predicted_span = predicted_spans[1], overlap_score= 0.82)

Reply:
Got it. I made the change in the code